mix/DebuggingStateWrapper.cpp

```diff
@@ -89,6 +89,7 @@ QStringList DebuggingStateWrapper::debugStack()
 	for (auto i: m_state.stack)
 		stack.append(QString::fromStdString(prettyU256(i)));
 
+	std::reverse(stack.begin(), stack.end());
```

Comment:
It is be more effecient to just walk the stack backwards using rcbegin/rcend in the for loop above